core/src/test/resources/diagnostics/lightblue-unhealthy-response.json

```diff
@@ -0,0 +1,29 @@
+{
+    "MongoCRUDController": {
+        "healthy": false,
+        "message": null,
+        "connectionsPerHost": "100",
+        "credentials": "[MongoCredential{mechanism=MONGODB-CR, userName='lightblue', source='admin', password=<hidden>, mechanismProperties={}}]",
+        "database": "data",
+        "maxResultSetSize": "15000",
+        "maxQueryTimeMS": "10000",
+        "mongoClient": "Mongo{options=MongoClientOptions{description='null', readPreference=primary, writeConcern=WriteConcern{w=majority, wTimeout=null ms, fsync=null, journal=null, readConcern=com.mongodb.ReadConcern@0, codecRegistry=org.bson.codecs.configuration.ProvidersCodecRegistry@dd9e4033, commandListeners=[], minConnectionsPerHost=0, maxConnectionsPerHost=100, threadsAllowedToBlockForConnectionMultiplier=5, serverSelectionTimeout=30000, maxWaitTime=120000, maxConnectionIdleTime=0, maxConnectionLifeTime=0, connectTimeout=10000, socketTimeout=0, socketKeepAlive=false, sslEnabled=false, sslInvalidHostNamesAllowed=false, alwaysUseMBeans=false, heartbeatFrequency=10000, minHeartbeatFrequency=500, heartbeatConnectTimeout=20000, heartbeatSocketTimeout=20000, localThreshold=15, requiredReplicaSetName='null', dbDecoderFactory=com.mongodb.DefaultDBDecoder$1@5bc5f738, dbEncoderFactory=com.mongodb.DefaultDBEncoder$1@642515bb, socketFactory=javax.net.DefaultSocketFactory@46fc1ccc, cursorFinalizerEnabled=true, connectionPoolSettings=ConnectionPoolSettings{maxSize=100, minSize=0, maxWaitQueueSize=500, maxWaitTimeMS=120000, maxConnectionLifeTimeMS=0, maxConnectionIdleTimeMS=0, maintenanceInitialDelayMS=0, maintenanceFrequencyMS=60000}, socketSettings=SocketSettings{connectTimeoutMS=10000, readTimeoutMS=0, keepAlive=false, receiveBufferSize=0, sendBufferSize=0}, serverSettings=ServerSettings{heartbeatFrequencyMS=10000, minHeartbeatFrequencyMS=500}, heartbeatSocketSettings=SocketSettings{connectTimeoutMS=20000, readTimeoutMS=20000, keepAlive=false, receiveBufferSize=0, sendBufferSize=0}}}",
```

Comment:
mongoClient was actually removed from the response because adding it to the response resulted in a connection leak.  It doesn't look like any of your tests are looking for this specifically, but perhaps we can remove it from the sample responses to avoid any confusion.  All of the interesting settings are in the mongoClientOptions anyway.